pkg/master/client_ca_hook_test.go

```diff
@@ -228,18 +228,18 @@ func TestWriteClientCAs(t *testing.T) {
 	}
 }
 
-func getFinalConfiMaps(client *fake.Clientset) (map[string]*api.ConfigMap, bool) {
-	ret := map[string]*api.ConfigMap{}
+func getFinalConfiMaps(client *fake.Clientset) (map[string]*corev1.ConfigMap, bool) {
```

Comment:
Nit: just noticed the typo here (`ConfiMaps`) but it is pre-existing so ...

Reply:
killed that 😉plz re-lgtm thx